multi-node/generic/controller-install.sh

```diff
@@ -49,6 +55,12 @@ function init_config {
             exit 1
         fi
     done
+
+    if [ $USE_CALICO = "true" ]; then
+        export K8S_NETWORK_PLUGIN="cni"
```

Comment:
Sorry, I guess it would be `K8S_VER=v1.2.3_coreos_cni.0` (unless this is being set elsewhere?)